### PR TITLE
User/semere/week 1

### PR DESCRIPTION
**Pull request by semere**
Opdrach points included

### CODE REVIEW

File: Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html [hunk 1]
```diff
@@ -7,9 +7,41 @@
...
 </head>
 <body>
-  <div class="ck-site">
-    <h1>Hello World!</h1>
-    <button class="btn btn-primary">Button</button>
+  <div class="compiler">
+    <!-- component --> 
+    <div class="header"> 
+      <h3 class="header__title header__title--green">Zaha Hadid</h3>
+      <p class="header__text">The first woman to receive the Pritzker Architecture Prize</p>
```
Comment (Marcelvbeek) [inline Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html]: This is my first example comment. Great code!

File: Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html [hunk 2]
```diff
@@ -7,9 +7,41 @@
   <link rel="shortcut icon" type="image/png" href="/assets/images/favicon.png" />
 </head>
 <body>
-  <div class="ck-site">
-    <h1>Hello World!</h1>
-    <button class="btn btn-primary">Button</button>
+  <div class="compiler">
+    <!-- component --> 
+    <div class="header"> 
+      <h3 class="header__title header__title--green">Zaha Hadid</h3>
```
[the remark this answers is not in the document]
Comment (semere) [reply Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html:13]: Yes, I have removed it.

File: Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html [hunk 3]
```diff
@@ -7,9 +7,41 @@
...
+      <p class="header__text">The first woman to receive the Pritzker Architecture Prize</p>
+    </div>
+    <!-- end component-->
+    <div class="image">
+      <img src="app/assets/images/beethoven.jpg" class="image__frame">
+      <p class="image__text">The picture depict the image of the Hoxton square, London, England </p>
+    </div>
+    <div class="body">
```
[the remark this answers is not in the document]
Comment (semere) [reply Semere-Leerplan-Boilerplate/opdracht-week-1/app/index.html:21]: Change of class naming accepted.

File: Semere-Leerplan-Boilerplate/opdracht-week-1/app/styles/app.scss [hunk 4]
```diff
@@ -21,3 +21,7 @@
 // BETTER THAT .ck-component-child .ck-component-child2 - div.ck-component-child > div.ck-component-child2
 // .ck-component--modifier (the modifier)
 @import 'components/ck-site';
```
[the remark this answers is not in the document]
Comment (semere) [reply Semere-Leerplan-Boilerplate/opdracht-week-1/app/styles/app.scss:23]: Accepted.

File: Semere-Leerplan-Boilerplate/opdracht-week-1/app/styles/components/_compiler.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
+.compiler {
+  .header {
```
[the remark this answers is not in the document]
Comment (semere) [reply Semere-Leerplan-Boilerplate/opdracht-week-1/app/styles/components/_compiler.scss:2]: Accepted and removed it.

**Pull request merged**